package-lock.json

```diff
@@ -10,16 +10,16 @@
       "integrity": "sha512-bfna97nmJV6nDJhXNPeEfxyMjWnt6+IjUAaDPiYRTBlm8L41n8nvw6UAqUCbvpFfU246gHPxW7sfWwqtF4FcYA==",
```

Comment:
Looks like you're using `npm` v5.x.x.  For this project, we should be using v6.x.x.  If you install and use `nvm` (Node Version Manager), it makes it pretty easy flipping back and forth.  A `package-lock.json` file shouldn't change without changes to `package.json`.

Reply:
I'm on node 10 and npm 6.4.1. I'll just take the package-lock from master.